go.mod

```diff
@@ -14,15 +14,16 @@ require (
 	github.com/cespare/cp v0.1.0
 	github.com/cloudflare/cloudflare-go v0.114.0
 	github.com/cockroachdb/pebble v1.1.2
-	github.com/consensys/gnark-crypto v0.14.0
+	github.com/consensys/gnark-crypto v0.16.0
+	github.com/crate-crypto/go-eth-kzg v1.3.0
 	github.com/crate-crypto/go-ipa v0.0.0-20240724233137-53bbb0ceb27a
 	github.com/crate-crypto/go-kzg-4844 v1.1.0
 	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc
 	github.com/deckarep/golang-set/v2 v2.6.0
 	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.0.1
 	github.com/donovanhide/eventsource v0.0.0-20210830082556-c59027999da0
 	github.com/dop251/goja v0.0.0-20230806174421-c933cf95e127
```

Comment:
Upstream are running an older version of this dependency https://github.com/ethereum/go-ethereum/blob/2cd5f22037f1c148a536b0d9b766365519c9cf60/go.mod#L23. Is there a good reason to maintain this diff?

Reply:
Hmm good question why we even have this diff in there. I'll check with git blame where this is coming from.

Reply:
Fascinating, this is coming from an upstream bump 2y ago https://github.com/ethereum-optimism/op-geth/commit/509cd428e9e8405f922fcbf8f73eed6792224d90

Reply:
And then 1y ago, they _downgraded_ that dependency again [here](https://github.com/ethereum/go-ethereum/commit/aadcb886753079d419f966a3bc990f708f8d1c3b#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6). This seems like an accident to me. Why would they downgrade in that PR?
I think it's safe to do either.

Reply:
Yes most likely an accident.

Reply:
Let's keep it as is for now, I think it's irrelevant really